test/repl.jl

```diff
@@ -137,6 +145,7 @@ if !is_windows() || Sys.windows_version() >= Sys.WINDOWS_VISTA_VER
     # yield make sure this got processed
     readuntil(stdout_read, "1+1")
     close(t)
+    readuntil(stdout_read, "\n\n")
```

Comment:
By the way, you can tell this line isn't strictly necessary for the test above, but I had to add it when the 7001 test was placed lower in the file: the reads are out of sync with the writes by this point in the integration test.